tests/unit/Travel/CommandTest.php

```diff
@@ -44,9 +45,9 @@ public function testCalculateTotal() : void
         $command = $this->createCommand($vehicle);
 
         $date = Date::now();
-        $command->addVehicleTravel(200, new TravelDetails($date, 'auv', 'Brno', 'Praha'));
-        $command->addVehicleTravel(220, new TravelDetails($date, 'auv', 'Praha', 'Brno'));
-        $command->addTransportTravel(Money::CZK(50000), new TravelDetails($date, 'a', 'Brno', 'Praha'));
+        $command->addVehicleTravel(200, new TravelDetails($date, Type::get(Type::CAR), 'Brno', 'Praha'));
```

Comment:
Co udělat spis `Type::CAR()`

Reply:
Zvažoval jsem to (u pár enumů to i máme), ale nakonec jsem ty factory metody nepřidával. Především kvůli tomu, že konstruování z konstatního stringu se dělá jen v testech, v aplikaci je vždy string proměnný.

Tohle tedy IMO dává smysl pro enumy, u kterých:
- Jsou nějaké hodnoty používané staticky (třeba `PaymentMethod::CASH` v pokladní knize)
- Je jich relativně málo - protože každá hodnota přidává další metodu